Aptoide/aptoide-client-v8
# Feature/asv 451 add session to the bi

**What does this PR do?**

   The app is now sending the session to the BI. After 30 min without using the app, we consider that the user is starting a new session when he starts the app again. 

**Database changed?**

   No

**Where should the reviewer start?**

- [x] gradle.properties
- [x] AptoideBiAnalytics.java
- [x] AptoideBiEventLogger.java
- [ ] ManagerPreferences.java
- [ ] AptoideBiAnalyticsTest.java
- [x] ManagedKeys.java

**How should this be manually tested?**
 
  Instead of the injected _sessionInterval_ use your own interval (in milliseconds) in AptoideBiEventLogger.java. Use Charles to make sure the session is not sent before the time interval. Run tests

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-451](https://aptoide.atlassian.net/browse/ASV-451)

**Code Review Checklist**
